app/views/miq_policy/_alert_list.html.haml

```diff
@@ -1,8 +1,10 @@
 #alert_list_div
-  - if x_active_tree == :alert_tree && @view
```

Comment:
No reason to delete this `if` either. 
We very much need it... if `@view` is nil, the code below will blow up.